decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
```

Comment:
- [ ] I would advise you to add this line `attr_accessor :nameable`, This property will automatically add getter and setter to the method you created.